pkg/server/server.go

```diff
@@ -2,59 +2,58 @@
 package server
 
 import (
-	"encoding/json"
 	"fmt"
+	"github.com/cmsgov/easi-app/pkg/handlers"
+	"github.com/okta/okta-jwt-verifier-golang"
 	"log"
 	"net/http"
-	"time"
+	"os"
+	"strings"
 )
 
-type status string
-
-const (
-	draft status = "DRAFT"
-)
+func isAuthenticated(authHeader string) bool {
```

Comment:
I think it's ok for this to live in `server`, but maybe we want it in a separate file from `server.go`, so this file can remain as handler setup with middleware implementation details.